development/otp-input-app/pages/otp-input/index.mdx

```diff
@@ -95,8 +96,17 @@ This example demonstrating the accepting the only number type values with suppor
 ```js path=../../components/otp-input/ReuseNumberWithPasteExample name=ReuseNumberWithPasteExample
 ```
 
+#### Email Verification
+We have made a email verification component using ReuseOTPInput. We have already made the email state for you. Just give your email state and add the functionality to this component and you can use it in your application.
```

Comment:
What is email state?

Reply:
since it is verification for email, I have made a state for email.

Reply:
Remove the part about email state in docs, as the user will just copy and use code, not use it as a component.

Reply:
I removed the email state documentation, I just added- "give your email and add functionality"